src/ObjectHelper.php

```diff
@@ -345,6 +345,7 @@ public function buildSubscription(stdClass $object)
             'billingState' => isset($object->billingState) ? $object->billingState : null,
             'startTime' => isset($object->startTimestamp) ? $object->startTimestamp : null,
             'endTime' => isset($object->endTimestamp) ? $object->endTimestamp : null,
+            'cancelReason' => isset($object->cancelReason->reason_code) ? $object->cancelReason->reason_code : null,
```

Comment:
Should this be `cancelReasonCode` since only the code is being set?

Reply:
My thinking here was that the gateway-agnostic thing to do would just be to call it a "cancel reason". Vindicia calls it a "cancel reason code", but that's on them. I figured if multiple gateways were doing this, "cancel reason" would make most sense, and the format of the data might vary wildly by gateway. What do you think? Does that make sense? Could I make it clearer somehow?

Reply:
I see your point! "cancel reason" it is!